.travis.yml

```diff
@@ -0,0 +1,9 @@
+sudo: false
+os:
+ - linux
+ - osx
+language: d
+d:
+ - dmd-beta
+ - dmd
+ - ldc
```

Comment:
> What about gdc?

I stopped testing/using it a long time ago, because one just runs into too many fixed bugs or missing features.